sources/views/top.js

```diff
@@ -47,4 +54,14 @@ export default class TopView extends JetView {
 	init() {
 		this.use(plugins.Menu, "top:menu");
 	}
+
+	urlChange(view, url) {
+		this.nameOfPageComponent = this.$$("nameOfPage");
+		const nameOfPageFromUrl = url[1].page;
```

Comment:
Instead of modifying some text from the URL segment you can refer to the selected option in the main menu